### PR TITLE
[DE] Update `Gameplay/Game modifier/Perfect`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
- [x] This article has a `no_native_review` tag, so please review the whole thing rather than just these updates

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Game_modifier/Perfect/de.md [hunk 1]
```diff
@@ -30,28 +28,27 @@ outdated_since: d8754759b3d191925fe04a3fd90689e2c5245566
 
 ## Beschreibung
 
-Die Mod **Perfect** ist eine [Spielmodifikation](/wiki/Gameplay/Game_modifier), die versucht, Spielern zu helfen, die die Note SS erreichen wollen. [Beatmaps](/wiki/Beatmap) gelten automatisch als nicht bestanden und werden neugestartet, sobald eine Note verfehlt wurde oder wenn bei einem [Hit-Objekt](/wiki/Gameplay/Hit_object) eine andere Punktzahl als 300 erzielt wurde.
+Die Mod **Perfect** ist eine [Spielmodifikation](/wiki/Gameplay/Game_modifier), die versucht, Spielern zu helfen, die die Note SS erreichen wollen. [Beatmaps](/wiki/Beatmap) gelten automatisch als nicht bestanden und werden neugestartet, sobald eine Note verfehlt wurde oder wenn bei einem [Hit-Objekt](/wiki/Gameplay/Hit_object) eine andere Beurteilung als GREAT erzielt wurde.
 
 Eine der folgenden Handlungen **wird dafür sorgen**, dass die Mod Perfect die Beatmap neustartet:
```
Comment (respektive) [inline wiki/Gameplay/Game_modifier/Perfect/de.md:33]:
```suggestion
Jede der folgenden Handlungen **wird dafür sorgen**, dass die Mod Perfect die Beatmap neustartet:
```

File: wiki/Gameplay/Game_modifier/Perfect/de.md [hunk 2]
```diff
@@ -30,28 +28,27 @@ outdated_since: d8754759b3d191925fe04a3fd90689e2c5245566
...
 
 - eine Note verfehlen
-- eine `100` oder `50` erzielen
+- ein OK oder MEH erzielen
 - einen Spinner nicht bestehen
-- Sliderbreaking
+- das [Ende eines Sliders](/wiki/Gameplay/Hit_object/Slider/Slidertail) verfehlen oder einen [Sliderbreak](/wiki/Gameplay/Judgement/Slider_break) hervorrufen
 
 Eine der folgenden Handlungen **wird nicht dafür sorgen**, dass die Mod Perfect die Beatmap neustartet:
```
Comment (respektive) [inline wiki/Gameplay/Game_modifier/Perfect/de.md:40]:
```suggestion
Jede der folgenden Handlungen **wird nicht dafür sorgen**, dass die Mod Perfect die Beatmap neustartet:
```

File: wiki/Gameplay/Game_modifier/Perfect/de.md [hunk 3]
```diff
@@ -7,8 +7,6 @@ tags:
   - PF
   - Spielmodifikation
 no_native_review: true
```
Comment (respektive) [inline wiki/Gameplay/Game_modifier/Perfect/de.md:9]:
```suggestion
```

**Review by respektive — CHANGES_REQUESTED** [review]
also cant leave suggestion cause github being github, but like 22 should be 
```
- Typ: Steigerung der Schwierigkeit
```

**Comment by The-Last-Cookie** [comment]
@respektive If you don't mind, I use `Erhöhung` instead of `Steigerung` for the mod type, since I have used this in every other mod article as well.

(and yes, I just checked each mod article for this incorrect type information. That must have been a copy-paste mistake.)

**Comment by respektive** [comment]
Ah, yeah that's good then. I didn't actually check other articles :see_no_evil:

**Review by respektive — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**